LogosLinuxInstaller.py

```diff
@@ -181,13 +180,13 @@ def parse_args(args, parser):
     if args.install_app:
         config.ACTION = installer.install
     elif args.run_installed_app:
-        config.ACTION = wine.run_logos
+        config.ACTION = wine.run_logos()
     elif args.run_indexing:
-        wine.run_indexing
+        wine.run_indexing()
     elif args.remove_library_catalog:
-        control.remove_library_catalog
+        control.remove_library_catalog()
     elif args.remove_index_files:
-        control.remove_all_index_files
+        control.remove_all_index_files()
```

Comment:
The functions should be used to set `config.ACTION` here but left uncalled because the rest of the runtime config is not yet setup. The defined action is then called later in `main`.

Reply:
Ah, okay. I got an error when running these and the solution was to add the parentheses. I will change all of these around.